sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Generated/ContainerRegistryRestClient.cs

```diff
@@ -349,10 +349,7 @@ internal HttpMessage CreateGetRepositoriesRequest(string last, int? n)
             {
                 uri.AppendQuery("n", n.Value, true);
             }
-            if (apiVersion != null)
-            {
-                uri.AppendQuery("api-version", apiVersion, true);
-            }
+            uri.AppendQuery("api-version", apiVersion, true);
```

Comment:
Couple of questions:
- I understand we have a new requirement to always include api-version in the path.  Could you please link the issue that this change is addressing?
- I didn't see a PR that updated data-plane libraries that included this change ... could you share that with me so I'll have greater context here?
Thanks!

Reply:
@m-nash and I discussed this separately in the context of this PR: https://github.com/Azure/azure-sdk-for-net/pull/26371